modules/src/core/ics23_commitment/merkle.rs

```diff
@@ -51,6 +51,22 @@ impl From<RawMerkleProof> for MerkleProof {
     }
 }
 
+impl From<MerkleProof> for RawMerkleProof {
```

Comment:
Question: What's a `RawMerkleProof` and when is it appropriate to use it vs. a "regular" `MerkleProof`?

Reply:
The `RawMerkleProof` is the type that is defined in ibc-go's protobuf files. `MerkleProof` here is our own domain type for it, and hence we should use it everywhere in our codebase except when it's time to interact with a gRPC endpoint (where we convert our `MerkleProof` back into a `RawMerkleProof` to be sent on the wire)